snuba/web/__init__.py

```diff
@@ -4,14 +4,16 @@
 
 
 class RawQueryException(Exception):
```

Comment:
I am fine with changing this to `QueryException`, as long as `RawQueryResult` also becomes `QueryResult`. I agree that it doesn't matter whether it comes specifically from this function or not.

> It feels to me that the code that catches it now, does not care whether this comes from raw_query.

Yes, in the literal sense (those calling code doesn't care if it was this function or some other function that threw the exception) but I do think that ideally the calling code would receive the original exception, rather than this wrapper, so I do think most (non-web) callers care in the sense that they'd rather this exception not exist in the first place.

I still don't like this design but I do think this is a bit more palatable.

Reply:
Renamed this here: e06ab1c